### DISCUSSION TITLE
ctrl+enter on quick reblog

### DISCUSSION THREAD

**Discussion by MigratoryCoconuts** [AprilSylph/XKit-Rewritten#1192] [Suggestion Box]
Hi team! love the extension. this is just a quick one, but i often submit forms online by pressing ctrl+enter. I would love to be able to do the same in the quick-reblog popup, when the tags textbox has focus. thank you!

**Comment by marcustyphoon** [comment]
Me too! We've got this as issue #899.

I should probably take over that PR, or something, come to think of it. Not sure what happened to the person who was working on it.
